cloud_info_provider/providers/openstack.py

```diff
@@ -205,7 +205,8 @@ def build_image(self, image, share):
             )
             extra_attrs = {}
         if "ad:base_mpuri" in extra_attrs:
-            other_info["base_mpuri"] = extra_attrs["ad:base_mpuri"]
+            extra_attrs["base_mpuri"] = extra_attrs["ad:base_mpuri"]
```

Comment:
two keys with the same value in the same dict?

Reply:
Yes, it's a bit cumbersome, but the cloud-info-api and AppDB expect to find `base_mpuri` while the key name in glance is `ad:base_mpuri`. This should be gone as we get rid of AppDB